.github/workflows/ci.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
       with:
-        node-version: '16'
+        node-version: 'lts/*'
```

Comment:
I think we'd like the Browser Test environment to be in sync with Netlify, where the builds are still on Node 16.

Reply:
That makes sense, I'll revert this change.

Reply:
There should be a comment on this line explaining that.